service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ReviewCMOControllerAboutToStartTest.java

```diff
@@ -0,0 +1,103 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.OverrideAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.fpl.controllers.cmo.ReviewCMOController;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.model.common.dynamic.DynamicList;
+import uk.gov.hmcts.reform.fpl.model.common.dynamic.DynamicListElement;
+import uk.gov.hmcts.reform.fpl.model.order.CaseManagementOrder;
+import uk.gov.hmcts.reform.fpl.service.DocumentSealingService;
+import uk.gov.hmcts.reform.fpl.utils.TestDataHelper;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.enums.CMOStatus.SEND_TO_JUDGE;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.element;
+
+@ActiveProfiles("integration-test")
+@WebMvcTest(ReviewCMOController.class)
+@OverrideAutoConfiguration(enabled = true)
+public class ReviewCMOControllerAboutToStartTest extends AbstractControllerTest {
+
+    @MockBean
+    private DocumentSealingService documentSealingService;
+
+    protected ReviewCMOControllerAboutToStartTest() {
+        super("review-cmo");
+    }
+
+    @Test
+    void shouldReturnCorrectMapWhenMultipleCMOsReadyForApproval() {
+        DocumentReference order = TestDataHelper.testDocumentReference();
+        List<Element<CaseManagementOrder>> draftCMOs = List.of(
+            element(buildCMO("Test hearing 21st August 2020", order)),
+            element(buildCMO("Test hearing 9th April 2021", order)));
+
+        CaseData caseData = CaseData.builder().draftUploadedCMOs(draftCMOs).build();
+
+        AboutToStartOrSubmitCallbackResponse response = postAboutToStartEvent(asCaseDetails(caseData));
```

Comment:
for consistency i would add postAboutToStartEvent method that accepts caseData, there is already such ver for postAboutToSubmit